prisma/schema.prisma

```diff
@@ -26,4 +27,6 @@ model Movie {
   runtimeMins   Int
   createdAt     DateTime    @default(now())
   updatedAt     DateTime    @updatedAt
+  user          User?        @relation(fields: [userId], references: [id])
+  userId        Int?
```

Comment:
Nice use of an optional one to many, I also did the same to allow for a Movie to not 'need' a user connected to it on creation.

Reply:
Thanks. I tried it without and was getting an error on migration, this seemed to solve it.

Reply:
The same thing happened to me, I wonder if it's needed.

Reply:
Some of the other code I've seen this morning doesn't have it so I think not, but it solved my problem and I couldn't see any issue with having it as optional in the schema as checking can be enforced elsewhere if it's mandatory. You should checkout Marian's code for a nice way to make sure the forms are filled properly or get an error.